contracts/ElectionPractice.sol

```diff
@@ -0,0 +1,166 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title - How to use this?
+ * @dev - deploy contract to DEXON netowkr
+ * @dev - copy deployed address to https://dexon-workshop-vote.netlify.com/
+*/
+
+/**
+ * @title - In this Election Contract, it could take as many rounds of election.
```

Comment:
Should we add a section for the owner?
`- As the owner of this contract, you are able to ...`

Reply:
should be alright. I saw that in the dev section